### DISCUSSION TITLE
What graph database does GraphRAG utilize for persisting the knowledge graphs ? Thanks

### DISCUSSION THREAD

**Discussion by zytbeyond** [microsoft/graphrag#641] [Q&A] [answered]
What graph database does GraphRAG utilize for persisting the knowledge graphs ? Thanks

**Comment by maslenkovas** [comment]
From `graphrag/vector_stores/` it follows that they use LanceDB or Azure AI Search vectore stores. [graphrag/vector_stores](https://github.com/microsoft/graphrag/tree/main/graphrag/vector_stores)

**Comment by natoverse** [comment, marked as answer]
For the knowledge graph itself, we do not use a graph database - all outputs are stored in parquet flat files. The "graph" focus is on network machine learning versus storage and retrieval.
